openlcs/openlcs/settings.py

```diff
@@ -301,6 +301,16 @@
 # cache location settings
 REDIS_CACHE_LOCATION = os.environ.get('REDIS_CACHE_LOCATION',
                                       'redis://localhost:6379/1')
+
+try:
+    # pylint:disable=wildcard-import,unused-wildcard-import
+    parent_dir = os.path.abspath(os.path.dirname(__file__))
+    if parent_dir not in sys.path:
+        sys.path.append(parent_dir)
+    from settings_local import *  # noqa
+except ImportError:
+    pass
+
```

Comment:
Why would you move `CACHES` back to the end of this file, could you elaborate why it has to be done this way? As talked in pull 131, it's counter design to do it this way.

I think it's frustrating to see changes like this because we've discussed this in the MR and in the sync meeting and after the meeting for several times.

Reply:
@buweixin

Reply:
@rhyw 
Sorry to ignore this. Forgot about modify it. and will make changes ASAP.